internal/api/e2e/api/dashboard_test.go

```diff
@@ -13,7 +13,7 @@
 
 //go:build integration
 
-package e2e
```

Comment:
I´d be in favor of keeping e2e as package, this is easier to read in my opinion but maybe this is only me 

Like having ``internal/api/e2e`` and ``internal/client/e2e``

Reply:
we will create a client package just for the e2e tests, that's a bit weird don't you think ?

Reply:
aaah nevermind I thought client was standing for the cli but it is the api client. Nevermind fine